gen-apidocs/generators/html.go

```diff
@@ -163,7 +163,7 @@ func (h *HTMLWriter) WriteDefinition(d *api.Definition) {
 	fmt.Fprintf(f, "<H2 id=\"%s\">%s</H2>\n", linkID, nvg)
 	fmt.Fprintf(f, "<TABLE class=\"col-md-8\">\n<THEAD><TR><TH>Group</TH><TH>Version</TH><TH>Kind</TH></TR></THEAD>\n<TBODY>\n")
 	fmt.Fprintf(f, "<TR><TD><CODE>%s</CODE></TD><TD><CODE>%s</CODE></TD><TD><CODE>%s</CODE></TD></TR>\n",
-	            d.GroupDisplayName(), d.Version, d.Name)
+				d.GroupDisplayName(), d.Version, d.Name)
```

Comment:
nit: check that the indentations are correct throughout this file.

Reply:
... this is really about where TABs are expanded and how they are expanded. sigh.